src/fmcore/prompt_tuner/evaluator/llm_as_a_judge_boolean_evaluator.py

```diff
@@ -17,13 +15,13 @@
 from fmcore.mapper.llm_inference_mapper import LLMInferenceMapper
 
 
-class LLMAsJudgeBooleanEvaluator(BaseEvaluator[LLMAsAJudgeInput, LLMAsAJudgeBooleanOutput]):
+class LLMAsJudgeBooleanEvaluator(BaseEvaluator[Dict, bool]):
```

Comment:
its not obvious teh usage of dict .. please add a todo to improve pydoc after end to end testing there